auditlog/models.py

```diff
@@ -6,15 +6,18 @@
 from django.conf import settings
 from django.contrib.contenttypes.fields import GenericRelation
 from django.contrib.contenttypes.models import ContentType
+from django.core import serializers
 from django.core.exceptions import FieldDoesNotExist
 from django.db import DEFAULT_DB_ALIAS, models
 from django.db.models import Q, QuerySet
 from django.utils import formats, timezone
 from django.utils.encoding import smart_str
 from django.utils.translation import gettext_lazy as _
 
+from .history import FieldHistoryValueManagerMixin
 
-class LogEntryManager(models.Manager):
+
+class LogEntryManager(FieldHistoryValueManagerMixin, models.Manager):
```

Comment:
I took the approach of a Mixin here so that I could move the "historical state" references to a new file. My thought is that if you decide to not include the manager features, it will be easier for me to maintain the fork by having the distinct code in a separate file (or at least as much of it is possible.)